lms/djangoapps/instructor/management/commands/compute_grades.py

```diff
@@ -3,7 +3,7 @@
 django management command: dump grades to csv files
 for use by batch processes
 """
-from instructor.offline_gradecalc import offline_grade_calculation
+# from instructor.offline_gradecalc import offline_grade_calculation
```

Comment:
oh..actually i have used this management command for testing only. I will revert the changes.